README.md

```diff
@@ -90,8 +90,11 @@ Some commands you'll want to learn and what they do:
   * Displays all levels in the specified skill
 * git gud load <skill_name>
   * Begin the first level of the specified skill
+* git gud load -<level_name>
+  * Begin the specified level of the current skill
 * git gud load <skill_name> <level_name>
-  * Begin the specified level (it is optional to specify the level)
+  * Begin the specified level (specifying the level is optional)

```

Comment:
Since we have the other git gud load <skill_name> above, we don't need to say it's optional